lib/documentLoader.js

```diff
@@ -15,5 +15,6 @@ module.exports = async function documentLoader(url) {
       document: context
     };
   }
-  throw new Error(`${url} is not an authorized context url.`);
+  throw new Error(`vc-js documentLoader could not resolve the ${url}.` +
+    'You might need a custom documentLoader.');
```

Comment:
See also the general context loader pattern comment here:
https://github.com/digitalbazaar/vc-js/pull/33/files#r376083802

As is this text would be a bit confusing.  This handler would be run last, so saying you need a custom loader is odd if you are using one.  In theory a chain of loaders was tested already, so maybe this should just say no loader found the URL.  Suggesting solutions here rather in documentation seems odd, and may add to confusion.  Maybe hint it's a loader issue and be brief?
```suggestion
  throw new Error(`Document loader unable to load URL "${url}".`);
```

Reply:
I think using the module `vc-js` in the error helps the user quickly identify the issue. Also if a user is already passing in a `documentLoader` then wouldn't they be aware of the `documentLoader` parameter? Your error takes us almost right back to the original issue this PR is supposed to correct: the error needs specificity. I do like that your error at least mentions that a Document loader in so it's definitely better than the original.

Reply:
@iainphd this PR is intended to help you. So what error wording would work best for you?